docs/toc.md

```diff
@@ -68,20 +73,18 @@
 #### [Thread-Safe Collections](essentials/collections/thread-safeCollections.md)
 ##### [BlockingCollection Overview](essentials/collections/threadsafe/blockingCollectionOverview.md)
 ##### [When to Use a Thread-Safe Collection](essentials/collections/threadsafe/whentoUseaThread-SafeCollection.md)
-### [🔧 Common Language Runtime (CLR)](essentials/clr.md)
-### [🔧 Class Library overview](essentials/classlibrary.md)
-### [🔧 Working with base types](essentials/basetypes.md)
-### [🔧 Exception management in .NET Core](essentials/exceptions.md)
-## [.NET Command Line Interface reference](cli/index.md)
-# [Application scenarios](apis/index.md)
-## [🔧 Working with dates](apis/datetime/index.md)
 ## [Working with collections and data structures](apis/collections/index.md)
-### [How to: Add and Remove Items from a ConcurrentDictionary](essentials/collections/threadsafe/how toAddandRemoveItems.md)
+### [How to: Add and Remove Items from a ConcurrentDictionary](essentials/collections/threadsafe/howtoAddandRemoveItems.md)
 ### [How to: Add and Take Items Individually from a BlockingCollection](essentials/collections/threadsafe/howtoAddandTakeItems.md)
 ### [How to: Add Bounding and Blocking Functionality to a Collection](essentials/collections/threadsafe/howtoAddBoundingandBlocking.md)
 ### [How to: Use ForEach to Remove Items in a BlockingCollection](essentials/collections/threadsafe/howtoUseForEachtoRemove.md)
 ### [How to: Use Arrays of Blocking Collections in a Pipeline](essentials/collections/threadsafe/howtoUseArraysofBlockingCollections.md)
 ### [How to: Create an Object Pool by Using a ConcurrentBag](essentials/collections/threadsafe/howtoCreateanObjectPool.md)
+### [🔧 Common Language Runtime (CLR)](essentials/clr.md)
+### [🔧 Class Library overview](essentials/classlibrary.md)
+### [🔧 Working with base types](essentials/basetypes.md)
+### [🔧 Exception management in .NET Core](essentials/exceptions.md)
```

Comment:
This is also done, no? Remove the wrench? It should have the sub-topics like we have for the collections.